psga/s_aureus/bactopia_one.nf

```diff
@@ -0,0 +1,98 @@
+    path "local-single-sample/*/quality-control/summary/*_R2-final_fastqc.zip", emit: final_fastqc_r2
+    path "local-single-sample/*/quality-control/summary/*_R1-original_fastqc.zip", emit: original_fastqc_r1
+    path "local-single-sample/*/quality-control/summary/*_R2-original_fastqc.zip", emit: original_fastqc_r2
+    path "local-single-sample/*/mlst/default/blast/*blast.json", emit: mlst_json
+    path "local-single-sample/bactopia-tools/mlst/mlst/*/*.tsv", emit: mlst_tsv
+    path "local-single-sample/bactopia-tools/mlst/mlst/*/logs/mlst/nf-mlst.err", emit: mlst_err
+    path "local-single-sample/*/logs/assembly_qc/checkm.log", emit: checkm_log
+    path "local-single-sample/*/assembly/checkm/bins/*/genes.gff", emit: checkm_genes_gff
+    path "local-single-sample/*/assembly/checkm/bins/*/hmmer.tree.txt", emit: checkm_hmmer_tree
+    path "local-single-sample/*/assembly/checkm/checkm-results.txt", emit: checkm_results_txt
+    path "local-single-sample/logs/custom_dumpsoftwareversions/versions.yml", emit: software_versions
+    path "local-single-sample/software_versions.yml", emit: all_software_versions
+    path "local-single-sample/bactopia-tools/mykrobe/mykrobe/*/*.json", emit: mykrobe_json
+    path "local-single-sample/bactopia-tools/mykrobe/mykrobe/*/logs/mykrobe/nf-mykrobe.log", emit: mykrobe_log
+    path "local-single-sample/bactopia-tools/mykrobe/mykrobe/*/logs/mykrobe/versions.yml", emit: mykrobe_versions
+    path "local-single-sample/*/antimicrobial-resistance/*-protein-report.txt", emit: antimicrobial_protein_report
+    path "local-single-sample/*/antimicrobial-resistance/*-gene-report.txt", emit: antimicrobial_gene_report
+    path "local-single-sample/*/annotation/*.tsv", emit: annotation_tsv
+    path "local-single-sample/*/assembly/*.json", emit: assembly_json
+    path "quast_assembly.zip", emit: quast_assembly_zip
+    path "local-single-sample/*/variants/GCF_004153365/*.txt", emit: variants_txt_for_csv_file
+    path "sample-*-csv-files/annotation-summary.txt", emit: annotation_summary
+    path "local-single-sample/bactopia-tools/mlst/mlst/software_versions.yml", emit: mlst_versions
+
+  script:
+  """
+#!/usr/bin/env python
+import os
+import re
+import shutil
+
+with open('hello.txt', 'w') as outfile:
+    # Sample ID is not passed to script so get it from the sequence file names
+    sample_id = None
+    for line in os.listdir(os.getcwd()):
+        match = re.match('(.+)_1.fastq.gz', line)
+        if match:
+            sample_id = match.group(1)
+    if not sample_id:
+        raise Exception('Cannot get sample_id from input filenames')
+
+    bactopia_cmd = 'bactopia --R1 {sample_id}_1.fastq.gz --R2 {sample_id}_2.fastq.gz --sample {sample_id} --datasets /datasets/ --species "Staphylococcus aureus" --coverage 100 --genome_size median --outdir local-single-sample --max_cpus 10 -qs 1 --max_retry 1 --run_checkm'.format(
```

Comment:
It would be better to replace `local-single-sample` with `bactopia-output` for clarity.

Reply:
good idea. I'll do it on the next PR if there is time